src/resources/postcss/tickets-commerce/admin/orders/table.pcss

```diff
@@ -1,7 +1,7 @@
 /**
  * Event Tickets - Admin Orders Page
  *
- * @since TBD
+ * @since 5.13.0
```

Comment:
Why is this one different?

Reply:
Because it wasn't replaced during its release - since i found it i think i would fix it